packages/solarflare-theme/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+*.vsix
```

Comment:
Can this be moved to the top-level `.gitignore`?

Reply:
Done.